policyengine_us/variables/gov/states/ut/tax/income/credits/ut_at_home_parent_credit_eligibile.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_us.model_api import *
+
+
+class ut_at_home_parent_credit_eligibile(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Eligible for the Utah at-home parent credit"
+    definition_period = YEAR
+    defined_for = StateCode.UT
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(period).gov.states.ut.tax.income.credits.at_home_parent
+
+        one_parent_income_eligible = (
+            person("irs_employment_income", period)
+            + person("self_employment_income", period)
+        ) < p.parent_max_earnings
+        employment_income_eligible = (
+            tax_unit.sum(one_parent_income_eligible) > 0
```

Comment:
shouldn't this filter on head_or_spouse?

Reply:
how about create a new variable `is_parent`?

Reply:
i don't think that's necessary since we're limiting the multiplication to children and step-parents count, so heads and spouses will capture it. but could you add a legal code reference to the python file to ensure we adhere to it?